man/crypttab.xml

```diff
@@ -104,10 +104,14 @@
       see above and below.</para></listitem>
 
       <listitem><para>The key may be acquired via a PKCS#11 compatible hardware security token or
-      smartcard. In this case an encrypted key is stored on disk/removable media, acquired via
-      <constant>AF_UNIX</constant>, or stored in the LUKS2 JSON token metadata header. The encrypted key is
-      then decrypted by the PKCS#11 token with an RSA key stored on it, and then used to unlock the encrypted
-      volume. Use the <option>pkcs11-uri=</option> option described below to use this mechanism.</para></listitem>
+      smartcard. In this case a savedata used in unlock process is stored on disk/removable media, acquired via
+      <constant>AF_UNIX</constant>, or stored in the LUKS2 JSON token metadata header.</para>
+      <para>For RSA private key, the savedata is an encrypted key. The encrypted key is then decrypted by
+      the PKCS#11 token with an RSA key stored on it, and then used to unlock the encrypted volume.</para>
+      <para> For EC private key, the savedata is the generated public key in enrollment process. The public key
+      is then used to derive shared secret with the private key stored in the PKCS#11 token. Then the derived
+      shared secret is used to unlock the volume.</para><para>Use the <option>pkcs11-uri=</option> option
```

Comment:
you are inserting a docbook paragraph break here, but you hide this within the same line, please also insert an ascii paragraph break here